lib/ransack/adapters/mongoid/base.rb

```diff
@@ -33,6 +33,14 @@ def quote_column_name name
         end
 
         module ClassMethods
+          def _ransack_aliases
+            @_ransack_alises ||= {}
```

Comment:
Definitely a typo, but that variable is always being accessed through it's reader, which is why the tests pass anyway.